_distutils_importer/__init__.py

```diff
@@ -0,0 +1,39 @@
+import sys
```

Comment:
How about calling this file `_distutils_override` and merge the functionality from `setuptools.distutils_patch`?